packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-modify-instance-group-by-name.ts

```diff
@@ -73,38 +79,25 @@ export namespace EmrModifyInstanceGroupByName {
     /**
      * Specific list of instances to be protected when shrinking an instance group.
      *
-     * @default No instancesToProtect
+     * @default - None
      */
     readonly instancesToProtect?: string[];
 
     /**
      * Specific list of instances to be terminated when shrinking an instance group.
      *
-     * @default No instancesToTerminate
+     * @default - None
      */
     readonly instancesToTerminate?: string[];
```

Comment:
What does none mean here? No instances will be terminated?

Reply:
yeah, no instances will be terminated when shrinking an instance group. was trying to avoid restating the parameter description but I can change it to say it one more time.